packages/grid/src/vaadin-grid.js

```diff
@@ -156,6 +156,7 @@ registerStyles('vaadin-grid', gridStyles, { moduleId: 'vaadin-grid-styles' });
  * Part name                  | Description
  * ---------------------------|----------------
  * `row`                      | Row in the internal table
+ * `collapsed-row`            | Collapsed row
```

Comment:
Let's also add new part names to the corresponding table in `vaadin-grid.d.ts`

Reply:
Oh, right. Added.